package.json

```diff
@@ -7,7 +7,7 @@
     "test": "test"
   },
   "scripts": {
-    "prepublish": "npm run build",
+    "preinstall": "npm run build",
```

Comment:
> Your work on this project is really helpful. So thank you.

Glad to hear! You're very welcome. 😃 

> I will send another pull request later.

Sounds good. Let me know if you have any questions!